optaplanner-docs/src/main/asciidoc/Chapter-Optimization_algorithms/Chapter-Optimization_algorithms.adoc

```diff
@@ -163,33 +154,31 @@ For example, try combining multiple algorithms together:
 [[powerTweaking]]
 == Power tweaking or default parameter values
 
-Many optimization algorithms have parameters which affect results and scalability.
+Many optimization algorithms have parameters that affect results and scalability.
 Planner applies __configuration by exception__, so all optimization algorithms have default parameter values.
-This is very similar to the Garbage Collection parameters in a JVM: most users have no need to tweak them, but power users do tweak them.
+This is very similar to the Garbage Collection parameters in a JVM: most users have no need to tweak them, but power users often do.
 
-The default parameter values are good enough for many cases (and especially for prototypes), but if development time allows, it can be well worth to power tweak them with the <<benchmarker,benchmarker>> for better results and scalability on a specific use case.
-The documentation for each optimization algorithm also declares its advanced configuration for power tweaking.
+The default parameter values are sufficient for many cases (and especially for prototypes), but if development time allows, it may be beneficial to power tweak them with the <<benchmarker,benchmarker>> for better results and scalability on a specific use case.
+The documentation for each optimization algorithm also declares the advanced configuration for power tweaking.
 
 [WARNING]
 ====
-The default value of parameters will change between minor versions, to improve them for most users (but not necessary for you). To shield yourself from these changes, for better or worse, always use the advanced configuration.
-This is not recommended.
+The default value of parameters will change between minor versions, to improve them for most users. The advanced configuration can be used to prevent unnecessary changes, however, this is not recommended.
 ====
 
 
 [[solverPhase]]
 == Solver Phase
 
-A `Solver` can use multiple optimization algorithms in sequence. *Each
-    optimization algorithm is represented by a solver ``**Phase**``.* There is never more than 1 `Phase` solving at the same time.
+A `Solver` can use multiple optimization algorithms in sequence. Each optimization algorithm is represented by a solver ``Phase``. There is never more than one `Phase` solving at the same time.
```

Comment:
Hmm, not a fan of using an important admonition here as it's not a pitfall or a side-effect important realization. It's the explanation of the main concept in this section and that's the important cardinality to realize. Can't we just keep the bold text here?